src/schema/datatypes/anat.yaml

```diff
@@ -0,0 +1,45 @@
+groups:
+# First group
+  - datatypes:
+      - anat
+    suffixes:
+      - T1w
+      - T2w
+      - T1rho
+      - T1map
+      - T2map
+      - T2star
+      - FLAIR
+      - FLASH
+      - PD
+      - PDmap
+      - PDT2
+      - inplaneT1
+      - inplaneT2
+      - angio
+    extensions:
+      - nii.gz
```

Comment:
uff... the difficult life of standardization folks.  In the light of the discussion within #152 I wonder if we should make them all explicitly start with `.` solely for consistency with terminology.  I do appreciate the fact that we do not do that for entities or suffixes (prefix with `_`) but oh well, what can I say -- standards are standards and it is beneficial to be consistent. What do you think @tsalo?

Reply:
Good catch. All done now.